src/NzbDrone.Core/MediaFiles/EpisodeImport/ImportDecisionMaker.cs

```diff
@@ -39,39 +44,46 @@ public class ImportDecisionMaker : IMakeImportDecision
             _mediaFileService = mediaFileService;
             _diskProvider = diskProvider;
             _videoFileInfoReader = videoFileInfoReader;
+            _sampleService = sampleService;
             _logger = logger;
         }
 
-        public List<ImportDecision> GetImportDecisions(List<string> videoFiles, Series series, bool sceneSource, QualityModel quality = null)
+        public List<ImportDecision> GetImportDecisions(List<string> videoFiles, Series series)
+        {
+            return GetImportDecisions(videoFiles, series, false, null);
+        }
+
+        public List<ImportDecision> GetImportDecisions(List<string> videoFiles, Series series, bool sceneSource, ParsedEpisodeInfo folderInfo)
         {
             var newFiles = _mediaFileService.FilterExistingFiles(videoFiles.ToList(), series);
 
             _logger.Debug("Analyzing {0}/{1} files.", newFiles.Count, videoFiles.Count());
 
-            return GetDecisions(newFiles, series, sceneSource, quality).ToList();
+            return GetDecisions(newFiles, series, sceneSource, folderInfo).ToList();
         }
 
-        private IEnumerable<ImportDecision> GetDecisions(IEnumerable<String> videoFiles, Series series, bool sceneSource, QualityModel quality = null)
+        private IEnumerable<ImportDecision> GetDecisions(List<String> videoFiles, Series series, bool sceneSource, ParsedEpisodeInfo folderInfo)
         {
+            var parseFolder = ParseFolder(videoFiles, series, folderInfo);
+
+            if (!parseFolder)
+            {
+                folderInfo = null;
+            }
+
             foreach (var file in videoFiles)
             {
                 ImportDecision decision = null;
 
                 try
                 {
-                    var localEpisode = _parsingService.GetLocalEpisode(file, series, sceneSource);
+                    var localEpisode = _parsingService.GetLocalEpisode(file, series, sceneSource, parseFolder ? folderInfo : null);
```

Comment:
Oops, the setting of `folderInfo` to null is wrong, we want to use it later to make sure we use the best quality (in the case the episode doesn't have the quality, like BR season packs).